common/flatpak-run.c

```diff
@@ -2340,9 +2340,13 @@ flatpak_run_setup_base_argv (FlatpakBwrap   *bwrap,
     "# Disable user pkcs11 config, because the host modules don't work in the runtime\n"
     "user-config: none\n";
 
+  if ((flags & FLATPAK_RUN_FLAG_NO_PROC) == 0)
+    flatpak_bwrap_add_args (bwrap,
+                            "--proc", "/proc",
```

Comment:
I have verified with `flatpak run --command=ls org.vim.Vim /proc` that `/proc` shows up fully populated when running an app; and with `find /proc` in an `apply_extra` script that it is missing while applying extra-data.